### PR TITLE
Redo screenshots #50

### PR DESCRIPTION
**Pull request by danmcatee**
Add screenshots to screenshots page and terminal screenshot #50

### CODE REVIEW

File: apps/index.html [hunk 1]
```diff
@@ -1,28 +1,17 @@
----
-layout: onecolumn
-title: Apps
-permalink: /apps/
-theme: blue-light_blue
-navitem: nav-apps
----
+--- layout: onecolumn title: Apps permalink: /apps/ theme: blue-light_blue navitem: nav-apps ---
```
Comment (plfiorini) [inline apps/index.html]: do not reformat the front matter

File: screenshots.html [hunk 2]
```diff
@@ -1,10 +1,4 @@
----
-layout: centered
-title: Screenshots
-permalink: /screenshots/
-theme: yellow-light_blue
-navitem: nav-screenshots
----
+--- layout: centered title: Screenshots permalink: /screenshots/ theme: yellow-light_blue navitem: nav-screenshots ---
```
Comment (plfiorini) [inline screenshots.html]: don't rewrite the front matter

File: apps/index.html (hunk 1 above)
Comment (plfiorini) [inline apps/index.html]: the first screenshot is badly named: `images/screenshots/screensho1.png` should be `images/screenshots/screenshot1.png`

**Comment by plfiorini** [comment]
There are a few issues that I highlighted in the comments

**Comment by danmcatee** [comment]
sry for that prettier went wild on that, fixed it

**Pull request merged**